src/articles/SearchField.js

```diff
@@ -1,41 +1,61 @@
-import { useState } from "react";
+import { useRef, useState } from "react";
 import strings from "../i18n/definitions";
 import * as s from "./SearchField.sc";
 import { ClearSearchButton } from "../components/allButtons.sc";
+import redirect from "../utils/routing/routing";
+import { toast } from "react-toastify";
 
 export default function SearchField({ api, query }) {
   const [searchTerm, setSearchTerm] = useState(query);
+  const [isFocused, setIsFocused] = useState(false);
+  const inputRef = useRef();
+
+  const handleFocus = () => setIsFocused(true);
+  const handleBlur = () => setIsFocused(false);
 
   function keyDownInSearch(e) {
     if (e.key === "Enter") {
-      console.log(searchTerm);
-      api.logUserActivity(api.SEARCH_QUERY, "", searchTerm, "");
-      window.location = `/articles?search=${searchTerm}`;
+      if (inputRef.current.value === "") {
+        toast.error("Can not search for empty string!");
```

Comment:
let's just not show anything here.
the user might have clicked on search by mistake, it's not nice to remind them that they are wrong. 
simply not do anything is probably the best thing here.